.github/pull_request_template.md

```diff
@@ -1,2 +1,45 @@
-<!-- Please refer to the related issue number -->
-closes #
+## Description
+
+<!-- Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change. -->
+
+Closes # (NO_ISSUE)
+
+## Type of change
+
+<!-- Please delete options that are not relevant. -->
+
+- [ ] Documentation
+- [ ] Code refactoring (Restructuring existing code w/o changing its external behavior)
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+## Previews
+
+<!-- Please paste screenshots to reveal changes in this PR. -->
+
+## Checklist
+
+- [ ] My code follows the style guidelines of this project.
+- [ ] I have performed a self-review of my own code.
+- [ ] I have commented on my code, particularly in hard-to-understand areas.
+- [ ] My changes generate no new CI warnings.
+- [ ] Any dependent changes have been merged and published in downstream modules.
```

Comment:
I cannot understand this one. Can you explain?

Reply:
Removed.